### ISSUE TITLE
All functionality of the screen is not operable through a keyboard (Tab) interface Widget Builder issue

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#803]
**Describe the bug**
Content, Resources and Display tab are not operable through a keyboard interface (tab) Widget Builder issue.

**To Reproduce**

1. Go to Widget Builder (http://localhost:9992/cm/app/?view=widgetbuilder&site=test4&mode=readonly&path=%2FSites%2Ftest4%2F&pathType=page)
2. Create a new widget and then select that widget.
3. Content, Resources and Display tab are not operable through a keyboard interface (tab) Admin issue.
4. And all the components inside these tabs also not accessible through Tab key. For e.g Javascript and CSS resources.

**Expected behavior**
All page elements should be operable through a keyboard interface (tab).

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Fixed with https://github.com/percussion/percussioncms/issues/782
